### PR TITLE
v0.3 preparations

### PR DESCRIPTION
**Pull request by netromdk**

### CODE REVIEW

File: slacker/slack_api.py [hunk 1]
```diff
@@ -43,3 +48,32 @@ def post(self, method, args = {}):
...
+      url = file_info['url_download']
+    else:
+      url = file_info['url_private_download']
+      headers['Authorization'] = 'Bearer {}'.format(self.__token)
+
+    self.__logger.debug('Downloading {} to {}'.format(url, folder))
+
+    res = requests.get(url, stream = True, headers = headers)
+    if res.status_code != 200:
+      self.__logger.warning('Unable to download {}'.format(url))
```
Comment (voidpirate) [inline slacker/slack_api.py]: Can we also log the status code and message from the response here?

File: slacker/slack_api.py [hunk 2]
```diff
@@ -43,3 +48,32 @@ def post(self, method, args = {}):
...
+      return
```
Comment (voidpirate) [inline slacker/slack_api.py]: Can return `None` here so client code can check and handle accordingly

File: slacker/slack_api.py (hunk 1 above)
Comment (netromdk) [reply slacker/slack_api.py]: Good point, I'll try to make it similar to `post`.

File: slacker/slack_api.py (hunk 2 above)
Comment (netromdk) [reply slacker/slack_api.py]: Good point.

File: slacker/commands/files/files_list_command.py [hunk 3]
```diff
@@ -65,16 +79,20 @@ def action(self, args = None):
...
       for f in files:
         if not args.total_size:
-          self.logger.info("  {:<50} {:>10}".format(f["name"], format_size(f["size"], binary = True)))
+          self.logger.info('  {:<50} {:>10}'.format(f['name'],
+                                                    format_size(f['size'], binary = True)))
         totalFiles += 1
-        totalSize += f["size"]
+        totalSize += f['size']
+        if args.download:
+          slack_api.download_file(f['id'], args.download)
```
Comment (voidpirate) [inline slacker/commands/files/files_list_command.py]: Can we catch the exception here and log the message as an error?

Comment (voidpirate) [reply slacker/commands/files/files_list_command.py]: Nevermind we are handling it in the `REPL` as a catch all

**Pull request merged**